test/functional/3rd-party/3rd-party-repo-manage-negative.bats

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+
+}
+
+test_teardown() {
```

Comment:
these are the default test_setup and test_teardown actions, so there is no need to add them